_posts/2017-08-10-using-AWS-API-with-cross-account-role(javascript).markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Using AWS With Cross Account Role (Javascript)"
+date:   2017-07-17
+categories: javascript aws-sdk
+author: Bryan Yu
+---
+
+Using AWS API With Cross Account Role (Javascript)
+
+This article highlights the preliminary steps of performing AWS API calls in Javascript for a cross account role without going into too much detail.
+
+Why would you want to do this? Because you are currently working on a project using Javascript’s AWS API and have absolutely no idea how to do stuff with cross accounts.
+
+Before starting, it is assumed that you have already configured the necessary AWS policies and groups for cross account access, if not look <a href="https://aws.amazon.com/blogs/security/how-to-enable-cross-account-access-to-the-aws-management-console/">here</a>.
```

Comment:
Add links directly to the relating text, rather than `[here]()` links

[AWS policies and groups for cross account access](https://aws.amazon.com/blogs/security/how-to-enable-cross-account-access-to-the-aws-management-console/)